config/config.yaml

```diff
@@ -114,11 +114,12 @@ calling:
           - somatic_tumor_medium
         filter: myfilter
 
-# Will be ignored if calc_consensus_reads is activated
+# If calc_consensus_reads is activated duplicates will be merged
 remove_duplicates:
   activate: true
 
-# Experimental: calculate consensus reads from PCR duplicates
+# Experimental: calculate consensus reads from PCR duplicates.
+# if 'remove_duplicates' is deactivate only overlapping pairs will be merged
```

Comment:
Please also add the revel setting to this template config (commented out).

Reply:
I guess you mean just adding `REVEL` as a plugin in the config?
This is done now.